api/ImageBitmapRenderingContext.json

```diff
@@ -0,0 +1,94 @@
+{
+  "api": {
+    "ImageBitmapRenderingContext": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ImageBitmapRenderingContext",
+        "support": {
+          "webview_android": {
+            "version_added": "55"
+          },
+          "chrome": {
+            "version_added": "55"
+          },
+          "chrome_android": {
+            "version_added": "55"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "46"
+          },
+          "firefox_android": {
+            "version_added": "46"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "42"
+          },
+          "opera_android": {
+            "version_added": "42"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
+          "deprecated": false
+        }
+      },
+      "transferImageBitmap() -> transferFromImageBitmap()": {
```

Comment:
Please use `"transferFromImageBitmap": {` as an identifier here.
Then, for Firefox, please express the alternative name, that used to exist, like this:

```json
"firefox": [
  {
   "version_added": "52"
  },
  {
    "version_added": "46",
    "version_removed": "52",
    "alternative_name": "transferImageBitmap"
  }
],
```

Same for `firefox_android`.

Reply:
OK - will do. Thanks